<h1 id="rosemary-mccloskey">Rosemary McCloskey</h1>
<p>github.com/rmcclosk | rmcclosk.math@gmail.com | 604-219-0157</p>
<h2 id="objective">Objective</h2>
<p>To launch a challenging and fulfilling career in software development.</p>
<h2 id="education">Education</h2>
<table>
<col width="70%" />
<col width="30%" />
<tbody>
<tr class="odd">
<td align="left">M.Sc. Bioinformatics Candidate<br />CIHR Strategic Training Program in Bioinformatics<br />90% Average<br />University of British Columbia</td>
<td align="right">Sep. 2015 - present</td>
</tr>
<tr class="even">
<td align="left">B.Sc. Mathematics Honours, Computer Science Minor<br />3.98 GPA<br />Simon Fraser University</td>
<td align="right">Sep. 2009 - June 2014</td>
</tr>
</tbody>
</table>
<h2 id="skills">Skills</h2>
<h3 id="programming-languages">Programming languages</h3>
<ul>
<li>Four years experience using C, Python, and R to develop research software.</li>
<li>Academic experience with Java for software engineering course project.</li>
<li>Markup languages: LaTeX, Markdown, HTML+CSS.</li>
</ul>
<h3 id="computing-environment">Computing environment</h3>
<ul>
<li>Five years experience in Linux, including bash scripting.</li>
<li>Familiar with development tools such as gdb, valgrind, Make, autotools.</li>
<li>Working knowledge of SQL.</li>
</ul>
<h3 id="mathematics-and-statistics">Mathematics and statistics</h3>
<ul>
<li>Strong formal mathematics background including calculus, linear algebra, and discrete mathematics.</li>
<li>Experience performing statistical tests, fitting models, applying Bayesian inference, and creating informative visualizations of data.</li>
</ul>
<h3 id="bioinformatics">Bioinformatics</h3>
<ul>
<li>Extensive experience processing genetic data and performing phylogenetic analyses.</li>
<li>Familiar with open source toolkits such as BioPython and Bioconductor.</li>
<li>Experience interfacing with public databases such as Genbank and the Cancer Genome Atlas.</li>
</ul>
<p> </p>
<h2 id="experience">Experience</h2>
<table>
<col width="72%" />
<col width="27%" />
<tbody>
<tr class="odd">
<td align="left">M.Sc. Candidate, Bioinformatics<br />BC Centre for Excellence in HIV/AIDS</td>
<td align="right">May. 2015 - present</td>
</tr>
</tbody>
</table>
<ul>
<li>Masters thesis project under the supervision of Dr. Art Poon (Medicine, University of British Columbia).</li>
<li>Developed a novel method.</li>
<li>Implemented method as open source C software (github.com/rmcclosk/netabc).</li>
</ul>
<table>
<col width="72%" />
<col width="27%" />
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br />Bioinformatics Training Program, UBC</td>
<td align="right">May. 2015 - present<br />Sep. 2014 - May 2015</td>
</tr>
</tbody>
</table>
<ul>
<li>Two four-month rotations under Dr. Ryan Morin (Molecular Biology and Biochemistry, Simon Fraser University) and Dr. Sara Mostafavi (Statistics and Medical Genetics, University of British Columbia).</li>
<li>Performed bioinformatic analyses of genomic and epigenetic data.</li>
</ul>
<table>
<col width="72%" />
<col width="27%" />
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br />BC Centre for Excellence in HIV/AIDS</td>
<td align="right">May - Dec. 2012</td>
</tr>
</tbody>
</table>
<ul>
<li>Performed extensive computational experimentation for publication-level bioinformatics research.</li>
<li>Produced two first-author peer-reviewed publications.</li>
<li>Junior developer on Python/Django software project Kive (github.com/cfe-lab/Kive).</li>
</ul>
<table>
<col width="72%" />
<col width="27%" />
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br />Mathematics Department, Simon Fraser University</td>
<td align="right">May - Aug. 2011</td>
</tr>
</tbody>
</table>
<ul>
<li>Four month co-op research term with Drs. Marni Mishna and Cedric Chauve (Mathematics, Simon Fraser University).</li>
<li>Implemented algorithms in Python for generation and analysis of combinatorial structures.</li>
</ul>
<h2 id="awards-and-scholarships">Awards and Scholarships</h2>
<table>
<col width="68%" />
<col width="31%" />
<tbody>
<tr class="odd">
<td align="left">Department of Statistics Award for Data Science<br />University of British Columbia<br /></td>
<td align="right">June 2015</td>
</tr>
<tr class="even">
<td align="left">Strategic Training Program in Bioinformatics Scholarship<br />Canadian Institutes of Health Research</td>
<td align="right">Sep. 2014 - present</td>
</tr>
<tr class="odd">
<td align="left">Walter C Koerner Fellowship<br />University of British Columbia</td>
<td align="right">Sep. 2014 - May. 2015</td>
</tr>
<tr class="even">
<td align="left">NSERC Canadian Graduate Scholarship - Masters<br />University of British Columbia</td>
<td align="right">Sep. 2014 - Aug. 2015</td>
</tr>
<tr class="odd">
<td align="left">Academic Scholarship, Basic Sciences<br />Canadian Conference on HIV/AIDS Research</td>
<td align="right">Apr. 2013</td>
</tr>
<tr class="even">
<td align="left">New Investigator Award<br />Conference on Retroviruses and Opportunistic Infections</td>
<td align="right">Mar. 2013</td>
</tr>
<tr class="odd">
<td align="left">Vice-President Research Undergraduate Student<br />Research Award<br />Simon Fraser University</td>
<td align="right">May - Aug. 2012</td>
</tr>
<tr class="even">
<td align="left">NSERC Undergraduate Student Research Award<br />Simon Fraser University</td>
<td align="right">May - Aug. 2011</td>
</tr>
<tr class="odd">
<td align="left">Undergraduate Open Scholarship<br />Simon Fraser University</td>
<td align="right">Sep. 2010 - Dec. 2013</td>
</tr>
<tr class="even">
<td align="left">Alumni Scholarship<br />Simon Fraser University</td>
<td align="right">Sep. 2010 - Sep. 2011</td>
</tr>
<tr class="odd">
<td align="left">Kennith M Strand Entrance Scholarship<br />Simon Fraser University</td>
<td align="right">Sep. 2009 - Apr. 2010</td>
</tr>
</tbody>
</table>
<h2 id="publications">Publications</h2>
<p></p>
<table>
<col width="100%" />
<tbody>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn2" class="footnoteRef" id="fnref2"><sup>2</sup></a></span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn3" class="footnoteRef" id="fnref3"><sup>3</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn4" class="footnoteRef" id="fnref4"><sup>4</sup></a></span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn5" class="footnoteRef" id="fnref5"><sup>5</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn6" class="footnoteRef" id="fnref6"><sup>6</sup></a></span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn7" class="footnoteRef" id="fnref7"><sup>7</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn8" class="footnoteRef" id="fnref8"><sup>8</sup></a></span></td>
</tr>
</tbody>
</table>
<h2 id="oral-presentations">Oral Presentations</h2>
<table>
<col width="100%" />
<tbody>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn9" class="footnoteRef" id="fnref9"><sup>9</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn10" class="footnoteRef" id="fnref10"><sup>10</sup></a></span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn11" class="footnoteRef" id="fnref11"><sup>11</sup></a></span></td>
</tr>
</tbody>
</table>
<h2 id="poster-presentations">Poster Presentations</h2>
<table>
<col width="100%" />
<tbody>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn12" class="footnoteRef" id="fnref12"><sup>12</sup></a></span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><a href="#fn13" class="footnoteRef" id="fnref13"><sup>13</sup></a></span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><a href="#fn14" class="footnoteRef" id="fnref14"><sup>14</sup></a></span></td>
</tr>
</tbody>
</table>
<div class="references">
<p>Joy, JB, RH Liang, T Nguyen, <strong>RM</strong> <strong>McCloskey</strong>, and AFY Poon, ‘Origin and Evolution of Human Immunodeficiency Viruses’, in <em>Global Virology I-Identifying and Investigating Viral Diseases</em> (Springer New York, 2015), pp. 587–611</p>
<p>Joy, JB, <strong>RM</strong> <strong>McCloskey</strong>, T Nguyen, RH Liang, Y Khudyakov, A Olmstead, and others, ‘Phylogenetic Estimation of the Temporal Spread of Hepatitis C Genotype 1a in North America’, 2015</p>
<p>———, ‘The Spread of Hepatitis C Virus Genotype 1a in North America: a Retrospective Phylogenetic Study’, <em>The Lancet Infectious Diseases</em>, 16 (2016), 124–29</p>
<p>Le, AQ, J Taylor, W Dong, <strong>R</strong> <strong>McCloskey</strong>, C Woods, R Danroth, and others, ‘Differential Evolution of a CXCR4-Using HIV-1 Strain in CCR5wt/Wt and CCR5<span class="math">Δ </span>32/<span class="math">Δ </span>32 Hosts Revealed by Longitudinal Deep Sequencing and Phylogenetic Reconstruction’, <em>Scientific Reports</em>, 5 (2015), 17607</p>
<p>Liang, RH, T Mo, W Dong, GQ Lee, LC Swenson, <strong>RM</strong> <strong>McCloskey</strong>, and others, ‘Theoretical and Experimental Assessment of Degenerate Primer Tagging in Ultra-Deep Applications of Next-Generation Sequencing.’, <em>Nucleic Acids Research</em>, 42 (2014), e98</p>
<p>Mann, JK, H Byakwaga, XT Kuang, AQ Le, CJ Brumme, P Mwimanzi, and others, ‘Ability of HIV-1 Nef to Downregulate CD4 and HLA Class I Differs Among Viral Subtypes’, <em>Retrovirology</em>, 10 (2013), 1</p>
<p><strong>McCloskey</strong>, <strong>R</strong>, Liang R, PR Harrigan, Z Brumme, and AFY Poon, ‘Reconstructing Transmitted HIV Genotypes by Phylogenetic Analysis of Serial Samples’, 2013</p>
<p><strong>McCloskey</strong>, <strong>RM</strong>, ‘Statistical Methods for Ancestral Reconstruction’, 2013</p>
<p><strong>McCloskey</strong>, <strong>RM</strong>, and AFY Poon, ‘Phylogenetic Estimation of Contact Network Parameters with Kernel-ABC’, 2016</p>
<p><strong>McCloskey</strong>, <strong>RM</strong>, R Liang, PR Harrigan, Z Brumme, and AFY Poon, ‘An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data’, 2013</p>
<p><strong>McCloskey</strong>, <strong>RM</strong>, RH Liang, PR Harrigan, ZL Brumme, and AFY Poon, ‘An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data’, <em>Journal of Virology</em>, 88 (2014), 6181–94</p>
<p><strong>McCloskey</strong>, <strong>RM</strong>, RH Liang, JB Joy, M Krajden, JS Montaner, PR Harrigan, and others, ‘Global Origin and Transmission of Hepatitis C Virus Nonstructural Protein 3 Q80K Polymorphism’, <em>The Journal of Infectious Diseases</em>, 211 (2015), 1288–95</p>
<p>Poon, AFY, RH Liang, JB Joy, T Nguyen, <strong>RM</strong> <strong>McCloskey</strong>, M Krajden, and others, ‘Global Origin and Transmission of HCV NS3 Substitution Q80K That Is Associated with Lower Simeprevir Susceptibility’, 2014</p>
<p>Poon, AFY, LW Walker, H Murray, <strong>RM</strong> <strong>McCloskey</strong>, PR Harrigan, and RH Liang, ‘Mapping the Shapes of Phylogenetic Trees from Human and Zoonotic RNA Viruses’, <em>PLoS ONE</em>, 8 (2013), e78122</p>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Joy, JB and others, ‘The Spread of Hepatitis C Virus Genotype 1a in North America: a Retrospective Phylogenetic Study’, <em>The Lancet Infectious Diseases</em> (2016).<a href="#fnref1">↩</a></p></li>
<li id="fn2"><p><strong>McCloskey</strong>, <strong>RM</strong> and others, ‘Global Origin and Transmission of Hepatitis C Virus Nonstructural Protein 3 Q80K Polymorphism’, <em>The Journal of Infectious Diseases</em> (2015).<a href="#fnref2">↩</a></p></li>
<li id="fn3"><p>Le, AQ and others, ‘Differential Evolution of a CXCR4-Using HIV-1 Strain in CCR5wt/Wt and CCR5<span class="math">Δ </span>32/<span class="math">Δ </span>32 Hosts Revealed by Longitudinal Deep Sequencing and Phylogenetic Reconstruction’, <em>Scientific Reports</em> (2015).<a href="#fnref3">↩</a></p></li>
<li id="fn4"><p>Joy, JB and others, ‘Origin and Evolution of Human Immunodeficiency Viruses’, in <em>Global Virology I-Identifying and Investigating Viral Diseases</em> (Springer New York, 2015).<a href="#fnref4">↩</a></p></li>
<li id="fn5"><p><strong>McCloskey</strong>, <strong>RM</strong> and others, ‘An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data’, <em>Journal of Virology</em> (2014).<a href="#fnref5">↩</a></p></li>
<li id="fn6"><p>Liang, RH and others, ‘Theoretical and Experimental Assessment of Degenerate Primer Tagging in Ultra-Deep Applications of Next-Generation Sequencing.’, <em>Nucleic Acids Research</em> (2014).<a href="#fnref6">↩</a></p></li>
<li id="fn7"><p>Poon, AFY and others, ‘Mapping the Shapes of Phylogenetic Trees from Human and Zoonotic RNA Viruses’, <em>PLoS ONE</em> (2013).<a href="#fnref7">↩</a></p></li>
<li id="fn8"><p>Mann, JK and others, ‘Ability of HIV-1 Nef to Downregulate CD4 and HLA Class I Differs Among Viral Subtypes’, <em>Retrovirology</em> (2013).<a href="#fnref8">↩</a></p></li>
<li id="fn9"><p><strong>McCloskey</strong>, <strong>RM</strong>, and AFY Poon, ‘Phylogenetic Estimation of Contact Network Parameters with Kernel-ABC’, 2016.<a href="#fnref9">↩</a></p></li>
<li id="fn10"><p><strong>McCloskey</strong>, <strong>R</strong> and others, ‘Reconstructing Transmitted HIV Genotypes by Phylogenetic Analysis of Serial Samples’, 2013.<a href="#fnref10">↩</a></p></li>
<li id="fn11"><p><strong>McCloskey</strong>, <strong>RM</strong>, ‘Statistical Methods for Ancestral Reconstruction’, 2013.<a href="#fnref11">↩</a></p></li>
<li id="fn12"><p>Joy, JB and others, ‘Phylogenetic Estimation of the Temporal Spread of Hepatitis C Genotype 1a in North America’, 2015.<a href="#fnref12">↩</a></p></li>
<li id="fn13"><p>Poon, AFY and others, ‘Global Origin and Transmission of HCV NS3 Substitution Q80K That Is Associated with Lower Simeprevir Susceptibility’, 2014.<a href="#fnref13">↩</a></p></li>
<li id="fn14"><p><strong>McCloskey</strong>, <strong>RM</strong> and others, ‘An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data’, 2013.<a href="#fnref14">↩</a></p></li>
</ol>
</div>
