<h1 id="rosemary-mccloskey">Rosemary McCloskey</h1>
<p>github.com/rmcclosk | rmcclosk.math@gmail.com | 604-219-0157</p>
<h2 id="objective">Objective</h2>
<p>To launch a challenging and fulfilling career in software development.</p>
<h2 id="education">Education</h2>
<table>
<col width="70%"/>
<col width="30%"/>
<tbody>
<tr class="odd">
<td align="left">M.Sc. Bioinformatics Candidate<br/>CIHR Strategic Training Program in Bioinformatics<br/>90% Average<br/>University of British Columbia</td>
<td align="right">Sep. 2015 - present</td>
</tr>
<tr class="even">
<td align="left">B.Sc. Mathematics Honours, Computer Science Minor<br/>3.98 GPA<br/>Simon Fraser University</td>
<td align="right">Sep. 2009 - June 2014</td>
</tr>
</tbody>
</table>
<h2 id="skills">Skills</h2>
<h3 id="programming-languages">Programming languages</h3>
<ul>
<li>Four years experience using C, Python, and R to develop research software.</li>
<li>Academic experience with Java for software engineering course project.</li>
<li>Markup languages: LaTeX, Markdown, HTML+CSS.</li>
</ul>
<h3 id="computing-environment">Computing environment</h3>
<ul>
<li>Five years experience in Linux, including bash scripting.</li>
<li>Familiar with development tools such as gdb, valgrind, Make, autotools.</li>
<li>Working knowledge of SQL.</li>
</ul>
<h3 id="mathematics-and-statistics">Mathematics and statistics</h3>
<ul>
<li>Strong formal mathematics background including calculus, linear algebra, and discrete mathematics.</li>
<li>Experience performing statistical tests, fitting models, applying Bayesian inference, and creating informative visualizations of data.</li>
</ul>
<h3 id="bioinformatics">Bioinformatics</h3>
<ul>
<li>Extensive experience processing genetic data and performing phylogenetic analyses.</li>
<li>Familiar with open source toolkits such as BioPython and Bioconductor.</li>
<li>Experience interfacing with public databases such as Genbank and the Cancer Genome Atlas.</li>
</ul>
<p> </p>
<h2 id="experience">Experience</h2>
<table>
<col width="72%"/>
<col width="27%"/>
<tbody>
<tr class="odd">
<td align="left">M.Sc. Candidate, Bioinformatics<br/>BC Centre for Excellence in HIV/AIDS</td>
<td align="right">May. 2015 - present</td>
</tr>
</tbody>
</table>
<ul>
<li>Masters thesis project under the supervision of Dr. Art Poon (Medicine, University of British Columbia).</li>
<li>Developed a Bayesian inference method using HIV sequence data to investigate properties of human contact networks.</li>
<li>Implemented method as open source C software (github.com/rmcclosk/netabc).</li>
</ul>
<table>
<col width="72%"/>
<col width="27%"/>
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br/>Bioinformatics Training Program, UBC</td>
<td align="right">Sep. 2014 - May 2015</td>
</tr>
</tbody>
</table>
<ul>
<li>Two four-month rotations under Dr. Ryan Morin (Molecular Biology and Biochemistry, Simon Fraser University) and Dr. Sara Mostafavi (Statistics and Medical Genetics, University of British Columbia).</li>
<li>Performed bioinformatic analyses of genetic and epigenetic data pertaining to cancer and Alzheimer's research.</li>
<li>Developed web interface using <em>shiny</em> framework to visualize changes in mutation frequency over time.</li>
</ul>
<table>
<col width="72%"/>
<col width="27%"/>
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br/>BC Centre for Excellence in HIV/AIDS</td>
<td align="right">May - Dec. 2012<br/>Jan. - Aug. 2014</td>
</tr>
</tbody>
</table>
<ul>
<li>Two eight-month co-op research assistantships with Dr. Art Poon (Medicine, University of British Columbia).</li>
<li>Performed extensive computational experimentation for publication-level bioinformatics research on HIV and hepatitis C phylogenetics.</li>
<li>Produced two first-author peer-reviewed publications with no formal training in biology or bioinformatics.</li>
<li>Junior developer on Python/Django software project for tracking bioinformatics pipelines (github.com/cfe-lab/Kive).</li>
</ul>
<table>
<col width="72%"/>
<col width="27%"/>
<tbody>
<tr class="odd">
<td align="left">Research Assistant<br/>Mathematics Department, Simon Fraser University</td>
<td align="right">May - Aug. 2011</td>
</tr>
</tbody>
</table>
<ul>
<li>Four month co-op research term with Drs. Marni Mishna and Cedric Chauve (Mathematics, Simon Fraser University).</li>
<li>Implemented algorithms in Python for generation and analysis of combinatorial structures.</li>
</ul>
<h2 id="awards-and-scholarships">Awards and Scholarships</h2>
<table>
<col width="68%"/>
<col width="31%"/>
<tbody>
<tr class="odd">
<td align="left">Department of Statistics Award for Data Science<br/>University of British Columbia<br/></td>
<td align="right">June 2015</td>
</tr>
<tr class="even">
<td align="left">Strategic Training Program in Bioinformatics Scholarship<br/>Canadian Institutes of Health Research</td>
<td align="right">Sep. 2014 - present</td>
</tr>
<tr class="odd">
<td align="left">Walter C Koerner Fellowship<br/>University of British Columbia</td>
<td align="right">Sep. 2014 - May. 2015</td>
</tr>
<tr class="even">
<td align="left">NSERC Canadian Graduate Scholarship - Masters<br/>University of British Columbia</td>
<td align="right">Sep. 2014 - Aug. 2015</td>
</tr>
<tr class="odd">
<td align="left">Academic Scholarship, Basic Sciences<br/>Canadian Conference on HIV/AIDS Research</td>
<td align="right">Apr. 2013</td>
</tr>
<tr class="even">
<td align="left">New Investigator Award<br/>Conference on Retroviruses and Opportunistic Infections</td>
<td align="right">Mar. 2013</td>
</tr>
<tr class="odd">
<td align="left">Vice-President Research Undergraduate Student<br/>Research Award<br/>Simon Fraser University</td>
<td align="right">May - Aug. 2012</td>
</tr>
<tr class="even">
<td align="left">NSERC Undergraduate Student Research Award<br/>Simon Fraser University</td>
<td align="right">May - Aug. 2011</td>
</tr>
<tr class="odd">
<td align="left">Undergraduate Open Scholarship<br/>Simon Fraser University</td>
<td align="right">Sep. 2010 - Dec. 2013</td>
</tr>
<tr class="even">
<td align="left">Alumni Scholarship<br/>Simon Fraser University</td>
<td align="right">Sep. 2010 - Sep. 2011</td>
</tr>
<tr class="odd">
<td align="left">Kennith M Strand Entrance Scholarship<br/>Simon Fraser University</td>
<td align="right">Sep. 2009 - Apr. 2010</td>
</tr>
</tbody>
</table>
<h2 id="publications">Publications</h2>
<p/>
<table>
<col width="100%"/>
<tbody>
<tr class="odd">
<td align="left"><span class="citation">Joy, JB, <strong>RM</strong> <strong>McCloskey</strong>, T Nguyen, RH Liang, Y Khudyakov, A Olmstead, M Krajden, JW Ward, PR Harrigan, JSG Montaner, and AFY Poon, &#8216;The Spread of Hepatitis C Virus Genotype 1a in North America: a Retrospective Phylogenetic Study&#8217;, <em>The Lancet Infectious Diseases</em> (2016).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>RM</strong>, RH Liang, JB Joy, M Krajden, JS Montaner, PR Harrigan, and AFY Poon, &#8216;Global Origin and Transmission of Hepatitis C Virus Nonstructural Protein 3 Q80K Polymorphism&#8217;, <em>The Journal of Infectious Diseases</em> (2015).</span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation">Le, AQ, J Taylor, W Dong, <strong>R</strong> <strong>McCloskey</strong>, C Woods, R Danroth, K Hayashi, MJ Milloy, AFY Poon, and ZL Brumme, &#8216;Differential Evolution of a CXCR4-Using HIV-1 Strain in CCR5wt/Wt and CCR5<span class="math">&#916;&#8198;</span>32/<span class="math">&#916;&#8198;</span>32 Hosts Revealed by Longitudinal Deep Sequencing and Phylogenetic Reconstruction&#8217;, <em>Scientific Reports</em> (2015).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation">Joy, JB, RH Liang, T Nguyen, <strong>RM</strong> <strong>McCloskey</strong>, and AFY Poon, &#8216;Origin and Evolution of Human Immunodeficiency Viruses&#8217;, in <em>Global Virology I-Identifying and Investigating Viral Diseases</em> (Springer New York, 2015).</span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>RM</strong>, RH Liang, PR Harrigan, ZL Brumme, and AFY Poon, &#8216;An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data&#8217;, <em>Journal of Virology</em> (2014).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation">Liang, RH, T Mo, W Dong, GQ Lee, LC Swenson, <strong>RM</strong> <strong>McCloskey</strong>, CK Woods, CJ Brumme, CK Ho, J Schinkel, JB Joy, PR Harrigan, and AFY Poon, &#8216;Theoretical and Experimental Assessment of Degenerate Primer Tagging in Ultra-Deep Applications of Next-Generation Sequencing.&#8217;, <em>Nucleic Acids Research</em> (2014).</span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation">Poon, AFY, LW Walker, H Murray, <strong>RM</strong> <strong>McCloskey</strong>, PR Harrigan, and RH Liang, &#8216;Mapping the Shapes of Phylogenetic Trees from Human and Zoonotic RNA Viruses&#8217;, <em>PLoS ONE</em> (2013).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation">Mann, JK, H Byakwaga, XT Kuang, AQ Le, CJ Brumme, P Mwimanzi, S Omarjee, E Martin, GQ Lee, B Baraki, R Danroth, <strong>R</strong> <strong>McCloskey</strong>, C Muzoora, DR Bangsberg, PW Hunt, PJR Goulder, BD Walker, PR Harrigan, JN Martin, Ndung&#8217;uT, MA Brockman, and ZL Brumme, &#8216;Ability of HIV-1 Nef to Downregulate CD4 and HLA Class I Differs Among Viral Subtypes&#8217;, <em>Retrovirology</em> (2013).</span></td>
</tr>
</tbody>
</table>
<h2 id="oral-presentations">Oral Presentations</h2>
<table>
<col width="100%"/>
<tbody>
<tr class="odd">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>RM</strong>, and AFY Poon, &#8216;Phylogenetic Estimation of Contact Network Parameters with Kernel-ABC&#8217;, HIV Dynamics and Evolution, Woods Hole (upcoming) (2016).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>R</strong>, Liang R, PR Harrigan, Z Brumme, and AFY Poon, &#8216;Reconstructing Transmitted HIV Genotypes by Phylogenetic Analysis of Serial Samples&#8217;, Canadian Conference on HIV/AIDS Research, Vancouver (2013).</span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>RM</strong>, &#8216;Statistical Methods for Ancestral Reconstruction&#8217;, Canadian Undegraduate Mathematics Conference, Montr&#233;al (2013).</span></td>
</tr>
</tbody>
</table>
<h2 id="poster-presentations">Poster Presentations</h2>
<table>
<col width="100%"/>
<tbody>
<tr class="odd">
<td align="left"><span class="citation">Joy, JB, <strong>RM</strong> <strong>McCloskey</strong>, T Nguyen, RH Liang, Y Khudyakov, A Olmstead, M Krajden, J Ward, PR Harrigan, JSG Montaner, and AFY Poon, &#8216;Phylogenetic Estimation of the Temporal Spread of Hepatitis C Genotype 1a in North America&#8217;, International AIDS Society Conference, Vancouver (2015).</span></td>
</tr>
<tr class="even">
<td align="left"><span class="citation">Poon, AFY, RH Liang, JB Joy, T Nguyen, <strong>RM</strong> <strong>McCloskey</strong>, M Krajden, JSG Montaner, and PR Harrigan, &#8216;Global Origin and Transmission of HCV NS3 Substitution Q80K That Is Associated with Lower Simeprevir Susceptibility&#8217;, HIV Dynamics and Evolution, Tucson (2014).</span></td>
</tr>
<tr class="odd">
<td align="left"><span class="citation"><strong>McCloskey</strong>, <strong>RM</strong>, R Liang, PR Harrigan, Z Brumme, and AFY Poon, &#8216;An Evaluation of Phylogenetic Methods for Reconstructing Transmitted HIV Variants Using Longitudinal Clonal HIV Sequence Data&#8217;, Conference on Retroviruses and Opportunistic Infection, Atlanta (2013).</span></td>
</tr>
</tbody>
</table>

